gui.py

```diff
@@ -125,71 +142,80 @@ def image_dialog(self):
             self, "QFileDialog.getOpenFileNames()", "",
             "All Files (*)", options=options)
         if file_name:
-            print(file_name)
+            global global_image_name
+            global global_user_email
+            global_image_name = file_name
```

Comment:
try adding two lines here:

file_name = str(file_name)
global_image_name.append(file_name)

If file_name is actually a list, try:
for entry in filename:
    entry = str(entry)
    global_image_name.append(entry)

Reply:
Hopefully this will fix the uploading errors, which will hopefully also resolve the processing errors